### PR TITLE
JIRA closed and open statuses are taken from JIRA, and are not requir…

### PR DESCRIPTION
**Pull request by udiraz2**
…ed in YML.

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -102,6 +102,11 @@ jira:
   closed-status:
     - Closed
     - Resolved
+  status-category-open-name:
+    - To Do
+    - In Progress
+  statusCategoryClosedName:
```
[the remark this answers is not in the document]
Comment (AvivCX) [reply src/main/resources/application.yml]: Thanks. 
let's discuss this before moving forward with the code

**Comment by AvivCX** [comment]
Thanks.
let's discuss this before moving forward with the code

File: src/main/resources/application.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (AvivCX) [reply src/main/resources/application.yml]: let's remove status category properties if it's not required

File: src/test/resources/application-jiramanualconfig.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
+  jira:
```
Comment (AvivCX) [inline src/test/resources/application-jiramanualconfig.yml]: please remove secrets from config file,

**Comment by udiraz2** [comment]
> please remove secrets from config file

Oops. 
Will remove.

**Pull request merged**